### PR TITLE
Reverse words in a given string using java 

### LINKED ISSUE #2122: Reverse words in a given string , python, cp
## 💥 Proposal

( The program takes a string and prints the words of the string in a reverse order 
The program's example

Input:
S = i like this program very much
Output: much very program this like i
)

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#reporting-new-issues)?

(YES)

### PR DESCRIPTION
**Pull request by RuchiGitHu**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Reverse text without reversing individual word in a String.

### Checklist

- [X] I've read the contribution guidelines.
- [X] I've checked the issue list before deciding what to submit.
- [X] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Fixed #2122 using java..

### CODE REVIEW

File: Java/cp/Reverse_words_in_String.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+		for (int i = 0; i <= n-1; i++) {
+			in[i] = scan.next();
+		}
+		System.out.println();
+		for (int i = n - 1; i >= 0; i--) {
+			System.out.print(in[i] + " ");
+		}
+   	}
+	  }
+/*
```
Comment (HarshCasper) [inline Java/cp/Reverse_words_in_String.java]: Format the Code properly

File: Java/cp/Reverse_words_in_String.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+* Given a string, we need to reverse it without reversing any word in it.
+* We can reverse each word of a string.
+* In other words, Reverse text without reversing individual word in a String.
+*/
+    import java.util.Scanner;
+    public class Reverse_String {
+  	public static void main(String[] args) {
```
Comment (HarshCasper) [inline Java/cp/Reverse_words_in_String.java]: Shift the main logic to some other function and use main() only for the driver code

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Comment by RuchiGitHu** [comment]
Hi @HarshCasper.
As I respect your time to review any PR. Please look into my efforts and let me know if there is any requirement for my code.

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Format the Code properly.

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**